src/Resources/components/ItemUnits/Form/index.js

```diff
@@ -0,0 +1,196 @@
+import { putItemUnit, postItemUnit } from '@codetanzania/emis-api-states';
+import { Button, Form, Input, Col, Row } from 'antd';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { notifyError, notifySuccess } from '../../../../util';
+import './styles.css';
+
+const { TextArea } = Input;
+
+// eslint-disable-next-line jsdoc/require-returns
+/**
+ * @class
+ * @name ItemUnitForm
+ * @description  Render form for creating a new Item Unit Of Measure
+ *
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+class ItemUnitForm extends Component {
+  static propTypes = {
+    isEditForm: PropTypes.bool.isRequired,
+    itemUnit: PropTypes.shape({
+      name: PropTypes.string,
+      type: PropTypes.string,
+      maxStockAllowed: PropTypes.string,
+      minStockAllowed: PropTypes.string,
+    }),
+    form: PropTypes.shape({ getFieldDecorator: PropTypes.func }).isRequired,
+    onCancel: PropTypes.func.isRequired,
+    posting: PropTypes.bool.isRequired,
+    types: PropTypes.arrayOf(PropTypes.string).isRequired,
+  };
+
+  static defaultProps = {
+    itemUnit: null,
+  };
+
+  // eslint-disable-next-line jsdoc/require-returns
+  /**
+   * @function
+   * @name onChangeColor
+   * @description  call back function to handle color change
+   *
+   * @param {Object} colors colors object
+   * @param {Object} colors.color current color
+   *
+   * @version 0.1.0
+   * @since 0.1.0
+   */
+  onChangeColor = ({ color }) => {
+    const {
+      form: { setFieldsValue },
+    } = this.props;
+    setFieldsValue({ color });
+  };
+
+  // eslint-disable-next-line jsdoc/require-returns
+  /**
+   * @function
+   * @name handleSubmit
+   * @description  call back function to handle submit action
+   *
+   * @param {Object} e event object
+   *
+   * @version 0.1.0
+   * @since 0.1.0
+   */
+  handleSubmit = e => {
+    e.preventDefault();
+
+    const {
+      form: { validateFieldsAndScroll },
+      itemUnit,
+      isEditForm,
+    } = this.props;
+
+    validateFieldsAndScroll((error, values) => {
+      if (!error) {
+        if (isEditForm) {
+          const updatedItem = Object.assign({}, itemUnit, values);
+          putItemUnit(
+            updatedItem,
+            () => {
+              notifySuccess('Unit of measure was updated successfully');
+            },
+            () => {
+              notifyError(
+                'Something occurred while updating unit of measure, please try again!'
```

Comment:
This should be `An error ....` and not something. Can you update for all remaining error messages?